packages/aws-cdk-lib/aws-rds/README.md

```diff
@@ -854,7 +854,32 @@ const instance = new rds.DatabaseInstance(this, 'Instance', {
 });
 ```
 
-**Note**: In addition to the setup above, you need to make sure that the database instance has network connectivity
+The following example shows enabling domain support for a Aurora databse cluster and creating an IAM role to access Directory Services.
+
+```ts
+declare const vpc: ec2.Vpc;
+const iamRole = new iam.Role(this, 'Role', {
+  assumedBy: new iam.CompositePrincipal(
+    new iam.ServicePrincipal('rds.amazonaws.com'),
+    new iam.ServicePrincipal('directoryservice.rds.amazonaws.com'),
+  ),
+  managedPolicies: [
+    iam.ManagedPolicy.fromManagedPolicyArn(this, 'RdsRole', 'arn:aws:iam::aws:policy/service-role/AmazonRDSDirectoryServiceAccess'),
```

Comment:
Why not use `fromAwsManagedPolicyName`?

Reply:
Why was I using the `fromManagerdPolicyArn`? 😂

Reply:
Let's fix the same in an integ test and an unit test so that contributors who see these codes will not misunderstand!

Reply:
https://github.com/aws/aws-cdk/pull/28559/files#diff-5faaee1dfd608a160551274db73518aeab8d44bd3351879b34941a052a0135f9R3991

https://github.com/aws/aws-cdk/pull/28559/files#diff-14e95b4529a923153b85040624674c00a266dbd44e28599755901a0ae7d89b6bR19

Reply:
I was using it elsewhere too.. Thanks for pointing that out. I've made the correction!